src/common/TrackedOp.cc

```diff
@@ -158,10 +158,10 @@ OpTracker::~OpTracker() {
 
 bool OpTracker::dump_historic_ops(Formatter *f, bool by_duration, set<string> filters)
 {
-  RWLock::RLocker l(lock);
```

Comment:
this would be nice to have if `tracking_enabled` were immutable. but we do expose the `set_tracking()` to the daemons. and they are calling it at run-time without protecting it with any extra locking.

Reply:
or, we can make `tracking_enabled` an atomic?

Reply:
already in this pr